### PR TITLE
Add docs for logging and plotting

### PR DESCRIPTION
**Pull request by yeukfu**
- Close https://github.com/rlworkgroup/garage/issues/2116. 
- Fix Trainer missed in api doc.

### CODE REVIEW

File: docs/user/logging_plotting.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+garage supports convenient and useful logging. garage uses [dowel](https://github.com/rlworkgroup/dowel)
+for logging. The `logger` supports many outputs, including
+
+- Std output
+- Text output
+- Csv output
+- TensorBoard output
+
+In garage's experiment, the `logger` will output to all these four.
```
Comment (haydenshively) [inline docs/user/logging_plotting.md:13]: Nitpick: ~all these four~ "all of these" or "all four of these"

File: docs/user/logging_plotting.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+-------------  -
+AverageReturn  1
+-------------  -
+2020-10-21 14:06:04 | [log_experiment] Logging messages:
+-------------  -
+AverageReturn  2
+-------------  -
+```
+
+To look output with TensorBoard, you can refer this [page](monitor_experiments_with_tensorboard).
```
Comment (haydenshively) [inline docs/user/logging_plotting.md:55]: "To look *at* output..." and "...refer *to* this..."

**Review by haydenshively — APPROVED** [review]
Looks good other than the small grammar issues I commented on

**Review by krzentner — APPROVED** [review]

**Comment by yeukfu** [comment]
@Mergifyio rebase

**Pull request merged**